### PR TITLE
Add JSON for v3 multistage experiment

### PR DESCRIPTION
**Pull request by AllegroFox**

### CODE REVIEW

File: messaging-experiments.yaml [hunk 1]
```diff
@@ -1116,3 +1116,171 @@
...
+        ratio: 1
+        groups:
+          - aboutwelcome
+        value: {}
+      - slug: treatment-import
+        ratio: 1
+        groups:
+          - aboutwelcome
+        value:
+          id: msw-import
```
Comment (punamdahiya) [inline messaging-experiments.yaml:1143]: @ncloudioj as discussed shortening message id to use prefix msw in v3 80 experiment

Comment (ncloudioj) [reply messaging-experiments.yaml:1143]: Looks good, thanks for the heads up!

File: messaging-experiments.json [hunk 2]
```diff
@@ -1629,5 +1629,250 @@
       "experimentDocumentUrl": "https://experimenter.services.mozilla.com/experiments/message-router-content-experiment-high-velocity-message-testing-cfr-to-pip/",
       "userFacingDescription": "Message Router Content Experiment: High Velocity Message Testing: CFR to PiP"
     }
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-V3-80-RELEASE",
+    "enabled": true,
+    "filter_expression": "env.version >= '80.' && env.version < '81.' && env.channel == 'beta'",
```
Comment (punamdahiya) [inline messaging-experiments.json:1636]: changing filterexpression to use default channel will help verify experiment with local build

"filter_expression": "env.version >= '80.' && env.version < '81.' && (env.channel == 'beta' || env.channel == 'default')"

Comment (AllegroFox) [reply messaging-experiments.json:1636]: done!

File: messaging-experiments.json [hunk 3]
```diff
@@ -1629,5 +1629,250 @@
       "experimentDocumentUrl": "https://experimenter.services.mozilla.com/experiments/message-router-content-experiment-high-velocity-message-testing-cfr-to-pip/",
       "userFacingDescription": "Message Router Content Experiment: High Velocity Message Testing: CFR to PiP"
     }
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-V3-80-RELEASE",
+    "enabled": true,
+    "filter_expression": "env.version >= '80.' && env.version <= '81.' && (env.channel == 'beta' || env.channel == 'default')",
```
Comment (punamdahiya) [inline messaging-experiments.json:1636]: to make it evaluate true in local 81, looks like filter expression  should be env.version >= '80.' && env.version < '82.' && (env.channel == 'beta' || env.channel == 'default')

**Pull request merged**